src/core/executor.js

```diff
@@ -4,13 +4,36 @@ const net = require("net");
 const Locale = require("../locales")("service.run");
 const {format} = require("util");
 
+/**
+ *
+ * Will print a debug message
+ * with the decorator `[DEBUG]`
+ *
+ * @param {string} message message to print
+ */
+function debug(message) {
+  logger.info(`[DEBUG]: ${message}`);
```

Comment:
What about adding chalk dependency and just add the color on the work DEBUG.
```suggestion
  logger.log('chalk.bold.blue("[DEBUG]:"), message, "\n");
```

Reply:
May use "MICROSERVICE OUTPUT" instead of "[DEBUG]"